roles/kubernetes-apps/ansible/tasks/main.yaml

```diff
@@ -5,6 +5,7 @@
   until: result.status == 200
   retries: 10
   delay: 6
+  when: inventory_hostname == groups['kube-master'][0]
```

Comment:
I guess , this will work for me as a delegate + run_once.  It will run less as well.  I'll try it shortly.